ansible/monitoring.yml

```diff
@@ -108,7 +108,16 @@
       copy:
         src: files/goss/goss.service
         dest: "/etc/systemd/system"
-        mode: 0750
+        mode: 0640
+      notify: reload systemd
+      when: systemd_run_dir.stat.exists and systemd_run_dir.stat.isdir
+      tags:
+        - goss-install
+
+    - name: Enable goss systemd service start on boot
+      systemd:
+        name: goss
+        enabled: yes
```

Comment:
so enabled implies started as well ?

Reply:
> so enabled implies started as well ?

Only after the next boot.  From systemctl manual:

> Enabling units should not be confused with starting (activating) units, as done by the start command. Enabling and starting
           units is orthogonal: units may be enabled without being started and started without being enabled. Enabling simply hooks the
           unit into various suggested places (for example, so that the unit is automatically started on boot or when a particular kind
           of hardware is plugged in).